### PR TITLE
Update french

### PR DESCRIPTION
**Pull request by damsfx**
Add missing tranlations

### CODE REVIEW

**Comment by mjauvin** [comment]
Thanks @damsfx !

File: lang/fr/lang.php [hunk 1]
```diff
@@ -67,7 +67,10 @@
         'delete_confirm' => 'Confirmez-vous la suppression de cet article ?',
         'delete_success' => 'Ces articles ont été supprimés avec succès.',
         'close_confirm' => 'L’article n’est pas enregistré.',
-        'return_to_posts' => 'Retour à la liste des articles'
+        'return_to_posts' => 'Retour à la liste des articles',
+        'posted_byline' => 'Posté dans :categories le :date.',
+        'posted_byline_no_categories' => 'Posté le :date.',
+        'date_format' => 'd M Y'
```
Comment (LukeTowers) [inline lang/fr/lang.php:73]: @mjauvin missing trailing comma

**Comment by mjauvin** [comment]
You need this for what reason @LukeTowers ? It's the end of the array.

**Comment by LukeTowers** [comment]
@mjauvin consistency and maintainability; especially in the language files. It's all too easy to add another element to an array and forget to re-add the missing trailing comma from the line. This should be a standard practice in Winter in my opinion.

**Comment by mjauvin** [comment]
Ok. Fixed in b7c8b4c

**Comment by LukeTowers** [comment]
Thanks @mjauvin!

**Pull request merged**